### DISCUSSION TITLE
Disabling warnings?

### DISCUSSION THREAD

**Discussion by garymh** [starship/starship#2709] [Q&A] [answered]
Hello! I work on a really large repo, so git is almost always timing out when I switch to the directory. I usually get:

```
[WARN] - (starship::utils): Executing command "git" timed out.
[WARN] - (starship::utils): You can set command_timeout in your config to a higher value to allow longer-running commands to keep executing.
```

Is it possible to disable these warnings?

**Comment by vladimyr** [comment, marked as answer]
One way would be to put `export STARSHIP_LOG=error` inside your shell profile.

https://github.com/starship/starship/blob/6817e7f6981d4cdd16367eec725e75bd6089d3a0/src/logger.rs#L45-L54

**Reply by garymh** [reply]
Perfect! Exactly what I was looking for. Thank you!

**Reply by jeffhandley** [reply]
Can the log level be set in the configuration file, or is setting it in the environment variable the only way to achieve this?

**Reply by chipbuster** [reply]
As of right now, it's only supported in the environment variable.
